### DISCUSSION TITLE
Confusing Backend Terminology

### DISCUSSION THREAD

**Discussion by xeruf** [solidusio/solidus#4647] [New Features or Ideas]
I find it rather confusing that the admin panel is called "backend", since I think of the server when hearing backend - so I was very confused to find backend stylesheets - and there is no explanation for what backend/frontend refers to in https://guides.solidus.io/developers/customizations/customizing-assets

It only cleared up somewhat when I found https://edgeguides.solidus.io/customization/customizing-the-backend/

Is there any chance that this can be renamed for more clarity, or at least mentioned in `vendor/assets/stylesheets/spree/backend/all.css` and in the documentation?

**Comment by waiting-for-dev** [comment]
Your concern makes sense, but that terminology is very rooted in Solidus. To begin with, we'd need to rename the `solidus_backend` gem to `solidus_admin`. We could do that, but TBH it's not a priority at this point.
